test/GithubApi.Gist.test.js

```diff
@@ -0,0 +1,62 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+require('dotenv').config();
+
+chai.use(chaiSubset);
+
+const instance = axios.create({
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
+const urlBase = 'https://api.github.com/gists';
+const promise = 'new Promise((resolve, reject) => {resolve(123)})';
+let createdGist;
+let getGist;
+let deleteGist;
+
+const gist = {
+  description: 'Creating a Gist',
+  public: true,
+  files: {
+    'Promise.js': {
+      content: promise
+    }
+  }
+};
+
+describe('Create and Delete a Gist', () => {
+  it('Create a Gist', async () => {
+    createdGist = await instance.post(`${urlBase}`, gist);
```

Comment:
Be carefull. This variable is being created inside this test and used on the others. This should be avoided because tests should be independent, which means that you should be able to run other tests without running this one. To solve this, take profit of mocha hooks like `describe` and `before`